PROSPECTUS.md

```diff
@@ -0,0 +1,25 @@
+# Kubernetes Community Days Portland Sponsorship Opportunies
+
+_Adapted from [the Bouvet Island prospectus](https://kubernetescommunitydays.org/events/2020-bouvet-island/sponsor/)_
+
+## Post-Event Social: Contact us (Limit 1)
+
+- Opportunity to sponsor a fully branded post-event social
+- 8 Tickets
+- Table presence at the venue
+- Logo on site and conference materials
+
+## Gold: #3,000 (Limit 3)
+
+- Logo on event materials and website
+- 8 tickets (max 2 at table at any time)
+- 5 minute “promo’ spot during talk intermissions
+- Table presence at the venue
+- Logo on site and conference materials
```

Comment:
Add:

- Thank you in opening and closing presentations
- Mention in all conference-related announcements
- Logo on banner at the conference site